consoles/sshVirtsh.pm

```diff
@@ -550,16 +550,16 @@ __END"
     return;
 }
 
-sub attach_to_running ($self, $args) {
-    my $name = ref($args) ? $args->{name} : $args;
+sub attach_to_running ($self, $args = undef) {
```

Comment:
Just curious, would this be working as well, without the other changes:
```suggestion
sub attach_to_running ($self, $args = {}) {
```

Reply:
If one calls the function like `$console->attach_to_running(compute_name())` then the `$args = {}` version breaks if `compute_name()` returns `undef` but this worked before converting the function to signatures. So I restored that compatibility. (The only incompatibility left is that one cannot call the function with more than two arguments but I suppose that's ok.)